pyspecdata/fourier/ft_shift.py

```diff
@@ -266,9 +265,14 @@ def ft_new_startpoint(self, axis, which_domain, value=None, nearest=False):
                         " for ft_clear_startpoints!!",
                     )
                 )
-        else:
-            value = orig_u - round(n_du) * du
+            if nearest:
+                value = round((value - orig_u) / du) * du + orig_u
+            else:
+                value = orig_u - round(n_du) * du
     self.set_ft_prop(axis, ["start", which_domain], value)
+    self.set_ft_prop(axis, [which_domain, "not", "aliased"], None)
+    if nearest is False:
+        self.set_ft_prop(axis, [other_domain, "not", "aliased"], True)
```

Comment:
using ft_new_startpoint to set FT property (has it been FTed with the new startpoint defined?)

Reply:
It's actually unsetting

Reply:
Add your missing the second part

Reply:
It's unsetting the ft property unless nearest is False in which case it is setting it to True

Reply:
above we set this property based on the startpoints we set either in the f domain or t domain

Reply:
this business with nearest is whether we have shifted by the nearest integral number of datapoints vs. the alternative which results in a phase dependent shift